pkgs/servers/home-assistant/custom-components/rct_power/package.nix

```
@@ -0,0 +1,36 @@
+{
+  lib,
+  fetchFromGitHub,
+  buildHomeAssistantComponent,
+  rctclient,
+}:
+
+buildHomeAssistantComponent rec {
+  owner = "weltenwort";
+  domain = "rct_power";
+  version = "v0.13.1";
+
+  src = fetchFromGitHub {
+    owner = "weltenwort";
+    repo = "home-assistant-rct-power-integration";
+    rev = "refs/tags/${version}";
+    hash = "sha256-a2cNlgYJImHek2cuvXokg6eHcAC/hM7tHa4bFdkzX9U=";
+  };
+
+  propagatedBuildInputs = [
+    rctclient
+  ];
+
+  #skip phases with nothing to do
+  dontConfigure = true;
+  dontBuild = true;
+  doCheck = false;
```

```suggestion
  doCheck = false; # no tests
```

The other phases are meaningless in `buildHomeAssistantComponent`. Check the code here: https://github.com/NixOS/nixpkgs/blob/master/pkgs/servers/home-assistant/build-custom-component/default.nix